### PR TITLE
Ret 2156 and ret 2157

### PR DESCRIPTION
**Pull request by gauravhmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-2156
https://tools.hmcts.net/jira/browse/RET-2157

### Change description ###

Delete ref data (file location and court worker)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/filelocation/FileLocationControllerTest.java [hunk 1]
```diff
@@ -215,6 +216,73 @@ void testUpdateFileLocationError() throws Exception {
         verify(fileLocationService, times(1)).updateFileLocation(ccdRequest.getCaseDetails().getAdminData());
     }
 
+    @Test
```
Comment (jthmcts) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/filelocation/FileLocationControllerTest.java:219]: Remove use of var in this class

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/filelocation/FileLocationController.java [hunk 2]
```diff
@@ -244,4 +244,43 @@ public ResponseEntity<CCDCallbackResponse> updateFileLocation(
...
+    })
+    public ResponseEntity<CCDCallbackResponse> deleteFileLocation(
+            @RequestHeader("Authorization") String userToken,
+            @RequestBody CCDRequest ccdRequest) {
+
+        if (!verifyTokenService.verifyTokenSignature(userToken)) {
+            return ResponseEntity.status(HttpStatus.FORBIDDEN.value()).build();
+        }
+
+        var adminData = ccdRequest.getCaseDetails().getAdminData();
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/filelocation/FileLocationController.java:280]: var

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/staff/CourtWorkerController.java [hunk 3]
```diff
@@ -145,4 +142,45 @@ public ResponseEntity<CCDCallbackResponse> updateCourtWorker(
...
+            @RequestHeader("Authorization") String userToken,
+            @RequestBody CCDRequest ccdRequest) {
+
+        log.info("/deleteCourtWorker");
+
+        if (!verifyTokenService.verifyTokenSignature(userToken)) {
+            return ResponseEntity.status(HttpStatus.FORBIDDEN.value()).build();
+        }
+
+        var adminData = ccdRequest.getCaseDetails().getAdminData();
```
Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/staff/CourtWorkerController.java:181]: var

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/filelocation/FileLocationController.java (hunk 2 above)
Comment (gauravhmcts) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/filelocation/FileLocationController.java]: removed

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/filelocation/FileLocationControllerTest.java (hunk 1 above)
Comment (gauravhmcts) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/filelocation/FileLocationControllerTest.java:219]: removed

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/staff/CourtWorkerController.java (hunk 3 above)
Comment (gauravhmcts) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/admin/staff/CourtWorkerController.java:181]: removed

**Review by jthmcts — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

**Pull request merged**